packages/hash/frontend/src/components/layout/PageSidebar/AccountEntityTypeList/EntityTypeMenu.tsx

```diff
@@ -0,0 +1,155 @@
+import { useRef, useState, VFC } from "react";
+import {
+  faEllipsis,
+  faLink,
+  faAdd,
+  faBookmark,
+  faTrash,
+  faFilter,
+} from "@fortawesome/free-solid-svg-icons";
+import pluralize from "pluralize";
+import { Box, IconButton, ListItemButton, Typography } from "@mui/material";
+import { FontAwesomeSvgIcon } from "../../../icons";
+import { Popover } from "../../../Popover";
+import { Link } from "../../../Link";
+
+type EntityTypeMenuProps = {
+  className: string;
+};
+
+const navItems = [
+  {
+    title: "Add to Bookmarks",
+    icon: faBookmark, // @todo-mui get a free icon that matches the design closely
+    onClick: () => {},
+  },
+  {
+    title: `Create new ${pluralize.singular("People")}`,
+    icon: faAdd,
+    href: "/",
+  },
+  {
+    title: "Copy Link to People",
+    icon: faLink,
+    onClick: () => {},
+  },
+  {
+    title: "Create filtered page",
+    icon: faFilter, // @todo-mui get a free icon that matches the design closely
+    onClick: () => {},
+  },
+  {
+    title: "Delete type",
+    icon: faTrash,
+    onClick: () => {},
+  },
+];
+
+export const EntityTypeMenu: VFC<EntityTypeMenuProps> = ({ className }) => {
+  const [open, setOpen] = useState(false);
```

Comment:
I think this should be open on first render
```suggestion
  const [open, setOpen] = useState(true);
```